firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -56,11 +62,16 @@ def test_notification_bar(self):
 
                 # Navigate to the unsafe page
                 # Check "ignore warning" link then notification bar's "not badware" button
-                self.marionette.navigate(unsafe_page)
-                # Wait for the DOM to receive events for about:blocked
-                time.sleep(1)
-                self.check_ignore_warning_button(unsafe_page)
-                self.check_not_badware_button(label, report_page)
+                # Only do this if feature supports it
+                if label is not None:
+                    self.marionette.navigate(unsafe_page)
+                    # Wait for the DOM to receive events for about:blocked
+                    time.sleep(1)
+                    self.check_ignore_warning_button(unsafe_page)
```

Comment:
Ok, you are right. The `label` variable is kinda misleading here. I had to triple check what this is. Would you mind updating the 3 instances of it to `button_property`? That is what it actually is. The label itself will be retrieved later via `get_property()`.